q2cli/builtin/tools.py

```diff
@@ -195,7 +196,104 @@ def peek(path):
         click.echo(metadata.format)
 
 
-@tools.command('inspect-metadata',
+@tools.command(name='cast-metadata',
+               short_help='Designate metadata column types.',
+               help='Designate metadata column types.'
+                    ' Supported column types can be found on docs.qiime2.org.'
```

Comment:
Awesome, this is shaping up well! One idea instead of putting docs.qiime2.org in here - you could interpolate the values of `_COLUMN_TYPES` into the string here.